src/blog/2022/04/flowforge-accepting-customers.md

```diff
@@ -0,0 +1,49 @@
+---
+title: FlowForge is accepting customers now
+subtitle: From today forward, we start accepting customers of our waitlist
+description: From today forward, we start accepting customers of our waitlist
+date: 2022-04-04
+authors: ["zeger-jan-van-de-weg"]
+---
+
+A year ago our CTO, Nick O'Leary, [introduced FlowForge](https://flowforge.com/blog/2021/04/first-deploy/).
+Since then major milestones have been achieved. As we grow as a company, important
+steps are taken. Today we make another very important step; we're accepting our
+first customers.
+
+<!--more-->
+
+A few weeks ago our product was nearly in a state where it could support customers,
+when that was the case, we [opened up our waitlist](...). The waitlist is growing
```

Comment:
```suggestion
when that was the case, we [opened up our waitlist](https://flowforge.com/blog/2022/02/announcing-flowforge-cloud/). The waitlist is growing
```

This link is to the original announce post - probably best place to link to the waitlist.